CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 - Add an artifact `opencensus-contrib-exemplar-util` that has helper utilities 
   on recording exemplars.
 - Reduce the default limit on `Link`s per `Span` to 32 (was 128 before).
+- Add Spring support for @Traced annotation and java.sql.PreparedStatements 
```

Comment:
nit backtick escape `@Traced` (and also code) mainly to prevent mentioning github users who might be named Traced